lavalink/lavalink.py

```diff
@@ -55,16 +55,16 @@ async def initialize(
         The hostname or IP address of the Lavalink node.
     password : str
         The password of the Lavalink node.
-    ws_port : int
+    port : Optional[int]
         The websocket port on the Lavalink Node.
```

Comment:
```suggestion
        The websocket port on the Lavalink Node.
        If not provided, it will use 80 for unsecured connections and 443 for secured.
```